package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ssense/restify-request-validator",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

Comment:
Could you just revert this change please ?
It will be incremented automaticcally during the npm versioning process.